docs-ref-conceptual/get-started-tutorial-3-use-variables.md

```diff
@@ -0,0 +1,326 @@
+$location="eastus"
+
+# Set your default output to none
+az config set core.output=none
+
+# Create a new Azure Key Vault returning the Key Vault ID
+$myNewKeyVaultID=$(az keyvault create --name $kvName --resource-group $rgName --location $location --query id --output tsv)
+echo "My new Azure Kev Vault ID is $myNewKeyVaultID"
+
+# Wait about 1 minute for your Key Vault creation to complete.
+
+# Create a new secret returning the secret ID
+$kvSecretName="<myKVSecretName>"
+$kvSecretValue="<myKVSecretValue>"
+$myNewSecretID=$(az keyvault secret set --vault-name $kvName --name $kvSecretName --value $kvSecretValue --query id --output tsv)
+echo "My new secret ID is $myNewSecretID"
+
+# Reset your default output to json
+az config set core.output=json
+```
+
+---
+
+## Get the contents of a JSON file and store it in a variable
+
+This next section is a "stretch task" for an onboarding tutorial. However, to manage Azure resources in development, stage and production environments, you often need to read a configuration file.
+
+Are you ready to stretch your Azure CLI skills? Create a JSON file containing the following JSON, or your file contents of choice. Save the text file to your local drive. If you're working in Azure Cloud Shell, use the `upload/download files` icon in the menu bar to store the text file in your cloud storage drive.
+
+```json
+{
+  "environments": {
+    "dev": [
+      {
+        "id": "1",
+        "kv-secretName": "dev1SecretName",
+        "status": "inactive",
+      },
+      {
+        "id": "2",
+        "kv-secretName": "dev2SecretName",
+        "status": "active"
+      }
+    ],
+    "stg": {
+      "id": "3",
+      "kv-secretName": "dev3SecretName"
+    },
+    "prod": {
+      "id": "4",
+      "kv-secretName": "dev4SecretName"
+    }
+  }
+}
+```
+
+Store the contents of your JSON file in a variable for further use in your Azure CLI commands. In this example, change `msdocs-tutorial.json` to the name of your file. Don't save the `echo` command in production-level scripts as the output is saved in your log file.
+
+# [Bash](#tab/bash)
+
+This Bash script was tested in [Azure Cloud Shell](/azure/cloud-shell/overview) and **depends on the Bash [jq](https://jqlang.github.io/jq/manual/) which must be installed in your environment.**
+
+```azurecli-interactive
+# Show the contents of a file in the console
+fileName=msdocs-tutorial.json
+cat $fileName | jq
+
+# Get a JSON dictionary object
+stgKV=$(jq -r '.environments.stg."kv-secretName"' $fileName)
+echo $stgKV
+
+# Filter a JSON array
+devKV=$(jq -r '.environments.dev[] | select(.status=="active") | ."kv-secretName"' $fileName)
+echo $devKV
+```
+
+Did you just receive a "jq command not found" error? This is because this script depends on the Bash [jq](https://jqlang.github.io/jq/manual/) command. [Install jq](https://jqlang.github.io/jq/download/) in your environment, or run this script in Azure Cloud Shell.
+
+# [PowerShell](#tab/powershell)
+
```

Comment:
Should this statement "This PowerShell script was tested in ..." before every command in previous tutorials?